e2e/helper_test.go

```diff
@@ -24,49 +24,52 @@ type CleanupContext struct {
 	CapacityAnnotations map[string]map[string]string
 }
 
-func execAtLocal(cmd string, input []byte, args ...string) ([]byte, []byte, error) {
-	var stdout, stderr bytes.Buffer
+// execAtLocal executes cmd.
+// If the cmd is failed, stdout and stderr is included in the error.
+// Therefore it is unnecessary to report them on caller.
+func execAtLocal(cmd string, input []byte, args ...string) (stdout []byte, stderr []byte, err error) {
```

Comment:
`stderr` is not used in 100% of cases, if my understanding is correct. I doubt we still need it.
When we find a case where we really want to handle stderr, then we can implement the function for that purpose.

(On the other hand, I imagine that it is super hard and boring to fix it... So this is just a comment. I will leave it to you.)

Reply:
My comment seems not good. 
I mean if this function returns an error, the error contains stderr, so you don't need to print the 2nd return value (stderr) explicitly, just print the error to show stderr instead.

Since this function still returns stdout and stderr even if an error happens, I assume the existing code never breaks.

I'll update the comment to express my intent more.

Reply:
Thank you for the clarification.

What I wanted to say is that we would be able to omit stderr from the returned values. Though it breaks the callers' code, no one is using the returned stderr because the callers now do not need to print it even when the error occurs.

If the signature looks like:
```Go
func execAtLocal(cmd string, input []byte, args ...string) (stdout []byte, err error)
```

then we can use it as follows
```Go
stdout, err := execAtLocal(foo, bar)
```

instead of
```Go
stdout, _, err := execAtLocal(foo, bar)
```

Reply:
Unfortunately, it can not. we use it.
https://github.com/topolvm/topolvm/pull/678/files#diff-e91d29c6d4c58503d9c2d72e2b04f1b70901a613e23008d1d1cfe8beff7038d4R214-R216